frontend_lib/src/container/RowWrappers/ListItemRowWrapper.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Link } from 'react-router-dom'
+
+import { PAGE } from '../../helper.js'
+import ListItemWrapper from '../../component/Lists/ListItemWrapper/ListItemWrapper.jsx'
+
+const ListItemRowWrapper = (props) => {
+  const contentTypeInfo = props.contentType.find(info => info.slug === props.originalType)
```

Comment:
This is a good practice to avoid js errors at l.12 if we don't find the type, for example when props.contentType is not yet initialized.
 
```suggestion
  const contentTypeInfo = props.contentType.find(type => type.slug === props.originalType) || { slug: '' }
```